docs/vectorizer.md

```diff
@@ -45,11 +45,19 @@ textual, data analysis, and semantic search:
 
 ## Select an embedding provider and set up your API Keys
 
-Vectorizer supports the following vector embedding providers:
+Vectorizer supports the following vector embedding providers as first-party integrations:
 - [Ollama](https://ollama.com/)
 - [Voyage AI](https://www.voyageai.com/)
 - [OpenAI](https://openai.com/)
 
+Additionally, through the [LiteLLM](https://litellm.ai) provider we support:
```

Comment:
can we add a a table of embedding providers to the front-page README (can do in separate PR).